x-pack/plugins/fleet/server/types/models/preconfiguration.ts

```diff
@@ -77,5 +88,9 @@ export const PreconfiguredAgentPoliciesSchema = schema.arrayOf(
         ),
       })
     ),
-  })
+  }),
+  {
+    // @ts-ignore
```

Comment:
Anyway to remove that ts-ignore? or a comment to explain why it's failing

Reply:
It's the conflicting type systems between Typescript and kbn-config-schema. I have no idea how to get them to cooperate with each other.

@jfsiii This was the reason why I initially removed the typedefs from the default policy constants, but it may be safer to ignore them here than to remove type annotations in the constants file

Reply:
They're not interchangeable, but they can cooperate. We do this in a few places elsewhere calling typeof on the kbn-config values.

I am not comfortable shipping an API endpoint with no/ignored types but that doesn't have to block this PR. We can file a ticket and do it as a bug fix.

If an ignore here allows us to keep the existing type (restore what's currently deleted), I'd like to do that before shipping this and address the rest in a follow up.